speedscale-operator 1.0.0

swtch1 opened the pull request:
Upgrade speedscale-operator chart to version 1.0.0.

samuelattwood commented:
Hey @swtch1 Your chart deploys fine and is ready to merge, but somehow it looks like you got some portworx files included in your commits which is not expected. We can just do some quick cleanup to your branch and it should be set

```
export PACKAGE=speedscale-operator
git reset fe208e30ffde8fa127d2d8f357456d1ad9df0497
git add packages/speedscale-operator
git commit -m "Make patch"
git checkout .
git clean -d -f
make charts
git add assets charts index.yaml
git commit -m "Make charts"
git push -f origin speedscale-operator-1.0.0
```

swtch1 commented:
@samuelattwood I agree, those portworx files are odd.  I did manually remove them for an earlier version of my change but decided to just follow the process to correct the repo.  It appears the portworx packages are not up to date with the validate step.  If you believe this is an upstream error that needs to be corrected I can wait and re-submit after the fix, or I'm happy to just run your steps which does not include `make validate`, though I suspect we'll see this again.

```
$ cd $(mktemp -d)
$  tmp.qi2ste5f git clone git@github.com:rancher/partner-charts.git
Cloning into 'partner-charts'...
remote: Enumerating objects: 10811, done.
remote: Counting objects: 100% (1116/1116), done.
remote: Compressing objects: 100% (688/688), done.
remote: Total 10811 (delta 412), reused 986 (delta 358), pack-reused 9695
Receiving objects: 100% (10811/10811), 14.67 MiB | 9.72 MiB/s, done.
Resolving deltas: 100% (4887/4887), done.
Updating files: 100% (9320/9320), done.
$  tmp.qi2ste5f cd partner-charts
$  partner-charts git:(main-source) make validate
Pulling in charts-build-scripts version https://github.com/rancher/charts-build-scripts.git@v0.2.2
charts-build-scripts-darwin => ./bin/charts-build-scripts
charts-build-scripts version v0.2.2 (e11e8fb)
INFO[0001] Generating charts and checking if Git is clean
...
➜  partner-charts git:(main-source) make clean
➜  partner-charts git:(main-source) git status
On branch main-source
Your branch is up to date with 'origin/main-source'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   assets/portworx/portworx-2.9.100.tgz
	modified:   charts/portworx/portworx/2.9.100/Chart.yaml
	modified:   charts/portworx/portworx/2.9.100/templates/_helpers.tpl
	modified:   charts/portworx/portworx/2.9.100/templates/portworx-crd.yaml
	modified:   charts/portworx/portworx/2.9.100/templates/portworx-csi.yaml

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	charts/portworx/portworx/2.9.100/Chart.yaml.orig

no changes added to commit (use "git add" and/or "git commit -a")
```
